lalrpop/src/lr1/codegen/parse_table.rs

```diff
@@ -749,6 +749,7 @@ impl<'ascent, 'grammar, W: Write> CodeGenerator<'ascent, 'grammar, W, TableDrive
                               format!("_: {}", self.phantom_data_type())];
 
         try!(self.out.write_pub_fn_header(self.grammar,
+                                          &Visibility::Pub(None),
```

Comment:
`pub(crate)` would suffice here, but I guess `pub` is fine too. In the future, it may start linting as a result though.